R/task-create.R

```diff
@@ -531,15 +531,32 @@ task_create_bulk_call <- function(fn, data, args = NULL,
 }
 
 
+task_export_envvars <- function(root, dst, envvars, inherit_envvars) {
```

Comment:
There's a similar issue with resources on a retried task where we might want to modify and might want to inherit.  This approach seems fine for now at least